carbon-design-system/carbon
# feat(IdPrefix): add component, docs, tests

Closes #12378 

Adds a new component and hook to enable consumers to prefix the autogenerated IDs placed on certain DOM elements

#### Changelog

**New**

- add `IdPrefix`, related docs and tests

#### Testing / Reviewing

- Ensure `IdPrefix` is exported
- Ensure the new stories for IdPrefix work and make sense
